proposals/lip-0014.md

```diff
@@ -632,4 +745,7 @@ Then `D` does not violate the protocol when using the initialization above.
 
 Let now `B_1` be the last block forged by `D` before crashing and `B_2` be the first block forged after performing the initialization above. By the above argument, we have  `B_1.height ≤ C.height + X`. Furthermore, as all blocks forged before time `T` have height at most `C.height + X`, we additionally obtain that  `B_1.maxHeightPreviouslyForged ≤ C.height + X`. Thus, if we set `B_2.maxHeightPreviouslyForged = C.height + X`, then property (i) is satisfied.
 
-The block at height `B_1.heightPrevoted` must receive a prevote by at least 35 of the honest delegates. Afterwards, these delegates will only prevote for blocks at heights larger than `B_1.heightPrevotes`. As at least one of these 35 delegates prevotes for `C`  (otherwise `C` can obtain at most 101 - 35 = 66 prevotes), we must have `B_1.heightPrevoted < C.height` because `C` is forged after `B_1`. Hence, we have `B_1.heightPrevoted < C.height ≤ B_2.heightPrevoted`.
+The block at height `B_1.heightPrevoted` must receive a prevote by >1/3 of the unchanged set of honest delegates. Afterwards, these delegates will only prevote for blocks at heights larger than `B_1.heightPrevotes`. As at least one of these delegates prevotes for `C`  (otherwise `C` cannot obtain >2/3 prevotes), we must have `B_1.heightPrevoted < C.height` because `C` is forged after `B_1`. Hence, we have `B_1.heightPrevoted < C.height ≤ B_2.heightPrevoted`.
```

Comment:
```suggestion
The block at height `B_1.heightPrevoted` must receive a prevote by >1/3 of the unchanged set of honest delegates. Afterwards, these delegates will only prevote for blocks at heights larger than `B_1.heightPrevotes`. As at least one of these delegates prevotes for `C` (otherwise `C` cannot obtain >2/3 prevotes), we must have `B_1.heightPrevoted < C.height` because `C` is forged after `B_1`. Hence, we have `B_1.heightPrevoted < C.height ≤ B_2.heightPrevoted`.
```